es-design-system/pages/molecules/es-footer.vue

```diff
@@ -0,0 +1,38 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div>
+        <h1>
+            EsFooter component
+        </h1>
+        <es-footer />
```

Comment:
Optional: Would be great if the docs page used a layout where the footer could go full-width so we can give it a proper layout test, but we can also do that later.